### DISCUSSION TITLE
New Birdnet-Analyzer version

### DISCUSSION THREAD

**Discussion by Fish31520** [mcguirepr89/BirdNET-Pi#1143] [Q&A]
Hi,

Few days ago a new version of Birdnet-Analyzer was released.

[https://github.com/kahst/BirdNET-Analyzer/releases](https://github.com/kahst/BirdNET-Analyzer/releases)

Is it automatically implemented in Birdnet-Pi or is there a tweak to do so (and is it really necessary/usefull ?).

Thanks !

Jean

**Comment by MueJosh** [comment]
Check out: 

[1](https://github.com/kahst/BirdNET-Analyzer/discussions/234)

[2](https://github.com/mcguirepr89/BirdNET-Pi/discussions/1105#discussioncomment-8424286)


I hope it's the same thing

**Reply by Fish31520** [reply]
Thanks for your help ans sorry for that late reply, I was AFK for few days.

Well, I'll have to check if it's what's described in the second link you gave me.

I'll let you know.
